tsdb/cursors/arrayvalues.gen.go.tmpl

```diff
@@ -0,0 +1,259 @@
+package cursors
```

Comment:
Sanity-checking: I don't see a new / changed `arrayvalues.gen.go` matching this file. Was the template just missing, or do we need to re-generate?

Reply:
The template was just missing. I've ran `go generate ./tsdb/cursors` with no changes.